python-binding/src/lib.rs

```diff
@@ -15,9 +15,9 @@ impl LinkContext {
         Ok(LinkContext { context })
     }
 
-    fn scan(&self) -> PyResult<Vec<String>> {
+    fn scan(&self, address: [u8; 5]) -> PyResult<Vec<String>> {
```

Comment:
What about adding a default value so that, in python, calling scan() scans with the default address?

This can be done with:
```
    #[args(address="[0xe7; 5]")]
```